apps/bfd-data-fda/Jenkinsfile

```diff
@@ -118,24 +118,17 @@ fi
                         if (ssmLookupSha256.isEmpty() || fdaDrugCodeSha256.trim() != ssmLookupSha256.trim()) {
                             dir('apps/bfd-data-fda') {
                                 def quietFlags = params.verbose_mvn_logging ? '' : '--quiet'
-                                sh "mvn ${quietFlags} clean install"
-                                sh """
-mvn ${quietFlags} exec:java \
--Dexec.mainClass="gov.cms.bfd.data.fda.utility.App" \
--Dexec.args="./src/main/resources/"
-"""
                                 sh 'echo "$CODEARTIFACT_ENDPOINT"'
-                                sh "mvn ${quietFlags} package"
 
                                 withCredentials([string(credentialsId: 'bfd-aws-account-id', variable: 'ACCOUNT_ID')]) {
                                     sh '''
-mvn deploy "-DaltDeploymentRepository=bfd-mgmt-bfd-mgmt::default::https://bfd-mgmt-${ACCOUNT_ID}.d.codeartifact.us-east-1.amazonaws.com/maven/bfd-mgmt/"
+mvn ${quietFlags} clean install deploy "-DaltDeploymentRepository=bfd-mgmt-bfd-mgmt::default::https://bfd-mgmt-${ACCOUNT_ID}.d.codeartifact.us-east-1.amazonaws.com/maven/bfd-mgmt/"
```

Comment:
I don't think you need install here. can you confirm and remove if not?

Reply:
yea we do, i tried it without install and it wouldnt generate the file.. :(.

Reply:
the problem is the ordering of the executions in the plugin section of the pom.xml. The one that you added back to download the file needs to appear before the one that copies resources. The reason why this fixed it is because it ran the build twice, once up to install and then again all the way to deploy, so the second build copied the resources from the first one.